src/utils/json.js

```diff
@@ -0,0 +1,36 @@
+export function formatJson(value, tabSpaces = 2) {
+  if (value == null) {
+    return null
+  }
+
+  try {
+    return JSON.stringify(value, null, tabSpaces)
+  } catch {
+    return null
+  }
+}
+
+export function parseJson(value) {
+  try {
+    return JSON.parse(value)
+  } catch {
+    return null
+  }
+}
+
+export function getJsonErrors(value) {
+  if (value == null || value == '') {
```

Comment:
I specifically don't want error on empty string because the underlying input is just a vuetify textarea, and when you enter text, then backspace all the way, you're left with an empty string, and I didn't want the user to still see an error.

I do agree that the behavior is a bit unexpected for a json utility method though, so I pulled it out into the 1 component that needs that specific logic (same for yaml)